.github/workflows/masterbot.yml

```diff
@@ -164,26 +164,36 @@ jobs:
   prerelease:
     timeout-minutes: 40
     name: Pre-release
-    # TODO: Uncomment job dependencies when we enable new versioning process.
+    # TODO: Uncomment when we enable new versioning process.
+    # env:
+    #   NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
     # needs:
     #   - 'lint-check'
     #   - 'unit-test'
     #   - 'e2e-atomic-test'
     #   - 'e2e-quantic-test'
     runs-on: ubuntu-latest
-    environment: prerelease
+    environment: 'Prerelease'
     steps:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
-      - uses: ./.github/actions/setup
+      - uses: actions/setup-node@v3
```

Comment:
At some point I also tested modifying the `setup` composite action to have an optional `build` boolean parameter and an optional `registry-url` parameter, but I reverted it because it's probably better to have less options.

Reply:
Essentially, we no longer want to build, because we want the new release process to build from scratch.